t4c/Dockerfile

```diff
@@ -53,9 +55,13 @@ USER root
 
 COPY setup_workspace.py /opt/setup/setup_t4c_client.py
 
-RUN chown techuser /opt/capella/capella.ini && \
-    echo '-DOBEO_LICENSE_SERVER_CONFIGURATION=$T4C_LICENCE_SECRET' >> /opt/capella/capella.ini && \
-    echo '-Duser.name=$T4C_USERNAME' >> /opt/capella/capella.ini
+RUN cat >> /opt/capella/capella.ini <<EOT && chown techuser /opt/capella/capella.ini
+-DOBEO_LICENSE_SERVER_CONFIGURATION=\$T4C_LICENCE_SECRET
+-Duser.name=\$T4C_USERNAME
```

Comment:
Is there any reason that the $ of `T4C_LICENCE_SECRET` and `T4C_USERNAME` are escaped? With this change we would now have 

```
-DOBEO_LICENSE_SERVER_CONFIGURATION=$T4C_LICENCE_SECRET
-Duser.name=$T4C_USERNAME
```

these two lines in the `capella.ini` instead of the actual licence secret and username.